Documentation/guides/OneDotNetSingleProject.md

```diff
@@ -67,16 +69,22 @@ The full list of defaults might be something like:
 
 ```xml
 <PropertyGroup>
-  <ApplicationVersion Condition=" '$(ApplicationVersion)' == '' ">1.0</ApplicationVersion>
-  <!-- Android only -->
-  <AndroidVersionCode Condition=" '$(AndroidVersionCode)' == '' ">1</AndroidVersionCode>
-  <!-- Apple platforms only -->
-  <AppleShortVersion Condition=" '$(AppleShortVersion)' == '' ">$(ApplicationVersion)</ApplicationVersion>
-  <AssemblyVersion Condition=" '$(AssemblyVersion)' == '' ">$(ApplicationVersion)</AssemblyVersion>
-  <FileVersion Condition=" '$(FileVersion)' == '' ">$(ApplicationVersion)</FileVersion>
+  <ApplicationVersion Condition=" '$(ApplicationVersion)' == '' ">1</ApplicationVersion>
+  <Version Condition=" '$(ApplicationDisplayVersion)' != '' ">$(ApplicationDisplayVersion)</Version>
```

Comment:
Should we check `Version` is empty before overwriting it? Users might be setting it via some build process like GitInfo.

Reply:
It's always set to `1.0.0` by the dotnet/sdk, so we don't know if users set it. We could maybe check if it equals `1.0.0`?